<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RENDEZVUE::ViNet</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="listView_source">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>641</width>
      <height>181</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget_image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>511</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_image">
     <attribute name="title">
      <string>Image</string>
     </attribute>
     <widget class="QLabel" name="image">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="image_bg">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>image_bg</zorder>
     <zorder>image</zorder>
    </widget>
    <widget class="QWidget" name="tab_result">
     <attribute name="title">
      <string>Result</string>
     </attribute>
     <widget class="QLabel" name="result">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="result_bg">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>result_bg</zorder>
     <zorder>result</zorder>
    </widget>
    <widget class="QWidget" name="tab_merge">
     <attribute name="title">
      <string>Result+Image</string>
     </attribute>
     <widget class="QLabel" name="merge">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="merge_bg">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>merge_bg</zorder>
     <zorder>merge</zorder>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>571</width>
      <height>721</height>
     </rect>
    </property>
    <property name="title">
     <string>Task</string>
    </property>
    <widget class="QPushButton" name="pushButton_update_job_tree">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resource.qrc">
       <normaloff>:/icons/image/icons/refresh.png</normaloff>:/icons/image/icons/refresh.png</iconset>
     </property>
    </widget>
    <widget class="CJobTree" name="treeWidget_job">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>551</width>
       <height>681</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_update_source_list">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>530</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/image/icons/refresh.png</normaloff>:/icons/image/icons/refresh.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1240</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEnsemble">
    <property name="title">
     <string>Ensemble</string>
    </property>
    <addaction name="actionSearch"/>
    <addaction name="actionConnect"/>
   </widget>
   <addaction name="menuEnsemble"/>
  </widget>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionCheck_for_updates">
   <property name="text">
    <string>Check for updates</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CJobTree</class>
   <extends>QTreeWidget</extends>
   <header>cjobtree.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
